.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release All SDKs
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  package-and-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Determine version
+        id: meta
+        run: |
+          ref="${GITHUB_REF}"
+          sha="${GITHUB_SHA}"
+          version="${ref#refs/tags/}"
+
+          echo "version=$version" >> $GITHUB_OUTPUT
+
+      - name: Package all SDKs
+        run: |
+          mkdir -p dist
+          for dir in *-sdk; do
+            if [ -d "$dir" ]; then
+              archive="dist/${dir}-${{ steps.meta.outputs.version }}.tar.gz"
+              echo "Packing $dir → $archive"
+              tar --exclude-vcs -czf "$archive" -C "$dir" .
+            fi
+          done
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: ${{ steps.meta.outputs.version }}
+          name: "SDK Release ${{ steps.meta.outputs.version }}"
+          overwrite: true
+          files: dist/*.tar.gz
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
```

Comment:
where does the `RELEASE_GITHUB_TOKEN` secret come from?

Reply:
it's a personal access token i set myself

Reply:
ah, ok great